frontend/src/views/TaskList.vue

```diff
@@ -95,6 +107,13 @@ const tasks = ref<TaskDetails[]>([
   }
   .topLevel {
     border-bottom: 2px solid #dddddd;
+    padding-bottom: 20px;
+  }
+  .groups:hover {
+    color: #555555;
+  }
+  .groups.active {
+    color: #6aa2b4; /* ここで希望の色を指定します */
```

Comment:
すでに設定してある変数を利用するようにしてください！
```suggestion
    color: $color-primary; /* ここで希望の色を指定します */
```